### PR TITLE
fix(modeling): AdaptiveLabelPositioningBehavior of labels before creation

### LINKED ISSUE #1211: Camunda Modeler crashes when pasting an Intermediate even with label onto a flow
__Describe the Bug__

Camunda modeler crashes when copying an intermediate event with a label onto a flow. 

The bug does not exist when I checkout to 5.0.3 release on Aug 22th (Commit hash: c9e9f002c9248c2b6fe2b3b1668447d01a69d054) but it exists on 5.0.4 release on Aug 29th (Commit hash: c01e64a16dadaf600d9911ce41b4266ce19d9e43)

See the GIF: 

Kapture 2019-10-02 at 17 59 35

__Steps to Reproduce__

1. Create an intermediate event
2. Give event a name (attach a label to it)
3. Copy the event
4. Paste the event onto a flow

__Expected Behavior__

App does not crash.

__Environment__

- See the bug description for the release info.

### PR DESCRIPTION
**Pull request by oguzeroglu**
Closes #1211

### CODE REVIEW

File: lib/features/modeling/behavior/AdaptiveLabelPositioningBehavior.js [hunk 1]
```diff
@@ -128,6 +128,10 @@ export default function AdaptiveLabelPositioningBehavior(eventBus, modeling) {
 
     var delta = substract(newLabelMid, labelMid);
 
+    if (!label.parent && label.labelTarget.parent) {
+      label.parent = label.labelTarget.parent;
```
Comment (philippfromme) [inline lib/features/modeling/behavior/AdaptiveLabelPositioningBehavior.js]: Monkey Patching 🐒 We don't want to do this. ⚠️

**Comment by oguzeroglu** [comment]
@philippfromme Modified my commit to ignore the labels on creation mode. Will do another enhancement PR to fix the label positioning issue.

**Comment by philippfromme** [comment]
> @philippfromme Modified my commit to ignore the labels on creation mode. Will do another enhancement PR to fix the label positioning issue.

Where's that PR?

**Comment by oguzeroglu** [comment]
It does not exist yet, waiting for this to get merged first. However this is the piece of code that'll do the work:

Screenshot 2019-10-04 at 16 07 49

**Pull request merged**